src/index.html

```diff
@@ -7,7 +7,287 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Miami</h1>
+    <header class="header">
+      <img src="images/phone.png"
+           alt=""
+           class="header__phone-img">
+      <input
+        type="checkbox"
+        class="header__nav-toggler" id="nav-toggler">
+      <label
+        class="header__nav-toggler-label" for="nav-toggler">
+      </label>
+      <div class="header__top">
+        <div class="header__phone">
+          <a class="header__phone" href="tel:+1 234 555-55-55">+1 234 555-55-55</a>
+        </div>
+        <a href="#">
+          <div class="logo header__logo"></div>
+        </a>
+        <div class="header__action">
+          <a href="#" class="header__link">Call to order</a>
+        </div>
+      </div>
+      <div class="header__caption">Miami Condo Kings</div>
+      <nav class="nav header__nav">
+        <ul class="nav__list">
+          <li class="nav__item nav__item-active">
+            <a href="#">Home</a>
+          </li>
+          <li class="nav__item">
+            <a href="#aboutUS" class="nav__link">About Us</a>
+          </li>
+          <li class="nav__item">
+            <a href="#gallery" class="nav__link">Gallery</a>
+          </li>
+          <li class="nav__item">
+            <a href="#consultation" class="nav__link">Consultation</a>
+          </li>
+          <li class="nav__item">
+            <a href="#testimonials" class="nav__link">Testimonials</a>
+          </li>
+          <li class="nav__item">
+            <a href="#contacts" class="nav__link">Contacts</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main>
+      <section class="container">
+        <div class="info" id="aboutUS">
+          <h1 class="info__heading">Miami best<br> Real estate
```

Comment:
Обычно мы либо все переносим на новую строку, либо все пишем в одной. Комбинированных вариантов нет
```suggestion
          <h1 class="info__heading">
            Miami best<br> Real estate
```